### DISCUSSION TITLE
Adding Zigbee features to Yale lock YRD446

### DISCUSSION THREAD

**Discussion by CanuckianOz** [Koenkk/zigbee2mqtt#7927] [Devices]
Hi @Koenkk,

I want to add more features to my Yale Assure YRD446. The below documentation details the data map for each feature. How do I take the data ID addresses in hex and add them to an external converter file? 

http://maxicom.net/public/Yale%20Locks%20ZigBee%20HA%20Integrators%20Guide%20RevC.pdf

Regards,
CanuckianOz

**Comment by Koenkk** [comment]
You can use https://github.com/Koenkk/zigbee-herdsman-converters/commit/de5a3d14c40e928b88784cce533b778dae22593f as an example on how to implement manufacturer specific features.
